src/App/Resources/AppResources.resx

```diff
@@ -2491,4 +2491,22 @@ Do you want to switch to this account?</value>
   <data name="LogInWithAnotherDevice" xml:space="preserve">
     <value>Log In with another device</value>
   </data>
+  <data name="LogInInitiated" xml:space="preserve">
+    <value>Log in initiated</value>
+  </data>
+  <data name="ANotificationHasBeenSentToYourDevice" xml:space="preserve">
+    <value>A notification has been sent to your device.</value>
+  </data>
+  <data name="PleaseMakeSureYourVaultIsUnlockedAndTheFingerprintPhraseMatchesOnTheOtherDevice" xml:space="preserve">
+    <value>Please make sure your vault is unlocked and the Fingerprint phrase matches on the other device.</value>
+  </data>
```

Comment:
Are the periods in the end needed? I don't usually put them

Reply:
ye the designs had periods. 
<img width="362" alt="image" src="https://user-images.githubusercontent.com/4648522/200605409-046ff3a7-4e78-4493-92ff-b4df9fd9ddfb.png">